nextflow.config

```diff
@@ -6,6 +6,24 @@
 ----------------------------------------------------------------------------------------
 */
 
+process {
+    withName: CUTADAPT {
+        ext.args = '-q 25'
```

Comment:
Here, I can add the arguments that I need to customise depending on the  project.